intake/tests/test_config.py

```diff
@@ -61,23 +62,24 @@ def test_conf():
 
 
 def test_conf_auth():
-    with temp_conf({'auth': {'class': 'intake.auth.secret.SecretAuth',
+    with temp_conf({'port': 5556,
```

Comment:
Changed not to be the default port?

Reply:
I was having issues where the port was already open in an unsecret way from other tests. Couldn't figure out how to make sure it was closed, so I just used a different port